.github/workflows/visual-regression.yml

```diff
@@ -0,0 +1,64 @@
+name: "Chromatic"
+
+on: push
+
+jobs:
+  cypress:
+    name: Run Cypress
```

Comment:
Can you please rename this to something like "Run Cypress for visual regression testing"?